[2023-04-30 09:46:18,094] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: multi_attack_dag.choose_attack manual__2023-04-30T09:46:13.115621+00:00 [queued]>
[2023-04-30 09:46:18,110] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: multi_attack_dag.choose_attack manual__2023-04-30T09:46:13.115621+00:00 [queued]>
[2023-04-30 09:46:18,111] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-04-30 09:46:18,113] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-04-30 09:46:18,113] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-04-30 09:46:18,140] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): choose_attack> on 2023-04-30 09:46:13.115621+00:00
[2023-04-30 09:46:18,151] {standard_task_runner.py:52} INFO - Started process 92 to run task
[2023-04-30 09:46:18,157] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'multi_attack_dag', 'choose_attack', 'manual__2023-04-30T09:46:13.115621+00:00', '--job-id', '7385', '--raw', '--subdir', 'DAGS_FOLDER/attack_dag.py', '--cfg-path', '/tmp/tmp4347rt45', '--error-file', '/tmp/tmpna6oiq9r']
[2023-04-30 09:46:18,158] {standard_task_runner.py:80} INFO - Job 7385: Subtask choose_attack
[2023-04-30 09:46:18,265] {task_command.py:369} INFO - Running <TaskInstance: multi_attack_dag.choose_attack manual__2023-04-30T09:46:13.115621+00:00 [running]> on host 873e5d3a4399
[2023-04-30 09:46:18,402] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=multi_attack_dag
AIRFLOW_CTX_TASK_ID=choose_attack
AIRFLOW_CTX_EXECUTION_DATE=2023-04-30T09:46:13.115621+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-30T09:46:13.115621+00:00
[2023-04-30 09:46:19,095] {python.py:173} INFO - Done. Returned value was: None
[2023-04-30 09:46:19,117] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=multi_attack_dag, task_id=choose_attack, execution_date=20230430T094613, start_date=20230430T094618, end_date=20230430T094619
[2023-04-30 09:46:19,174] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-04-30 09:46:19,289] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
